Source/artifacts/ArtifactsManager.js

```diff
@@ -50,4 +50,55 @@ export class ArtifactsManager {
         
         _boilerPlatesManager.get(this).createArtifactInstance('command', 'csharp', boilerPlate, destination, context);
     }
+    /**
+     * Create an event
+     * @param {string} name 
+     * @param {string} namespace 
+     */
+    createEvent(name, namespace) {
+        this._logger.info(`Creating event with name '${name}' and namespace '${namespace}'`);
+
+        let boilerPlate = _boilerPlatesManager.get(this).boilerPlatesByType('artifacts')[0];
+        let context = {
+            name: name,
+            namespace: namespace
+        };
+        let destination = process.cwd();
+        
+        _boilerPlatesManager.get(this).createArtifactInstance('event', 'csharp', boilerPlate, destination, context);
+    }
+    /**
+     * Create a read model
+     * @param {string} name 
+     * @param {string} namespace 
+     */
+    createReadModel(name, namespace) {
+        this._logger.info(`Creating read model with name '${name}' and namespace '${namespace}'`);
+
+        let boilerPlate = _boilerPlatesManager.get(this).boilerPlatesByType('artifacts')[0];
+        let context = {
+            name: name,
+            namespace: namespace
+        };
+        let destination = process.cwd();
+        
+        _boilerPlatesManager.get(this).createArtifactInstance('readModel', 'csharp', boilerPlate, destination, context);
+    }
+    /**
+     * Create an event
+     * @param {string} name 
+     * @param {string} namespace 
+     */
+    createAggregateRoot(name, namespace) {
```

Comment:
Namespace should be deducted. 
Remember; namespace is a C# feature - and we can figure out what it is. As mentioned in the issue: https://github.com/dolittle-tools/cli/issues/8

Lets walk through a bit on context and what the thinking has been.